local_archives/nwp/ecmwf/ecmwf_ens_stat_india.py

```diff
@@ -0,0 +1,70 @@
+import datetime as dt
```

Comment:
Do you mind adding a descirption at the top of this file. Im not xure what `ecmwf_ens_stat_india` means
ECWFM ensembles static values for india? What does static mean?

Reply:
If you look at the `@asset` declaration on line 35, there's an in-asset description which is surfaced in the dagster UI. Will that do, or do you want a docstring as well?

Reply:
I've moved the description to the docstring for the best of both worlds, hopefully!